src/test/scala/ai/privado/languageEngine/go/tagger/GoTaggingTestBase.scala

```diff
@@ -39,6 +41,7 @@ abstract class GoTaggingTestBase extends AnyWordSpec with Matchers with BeforeAn
   var inputDir: File   = _
   var outputFile: File = _
   val ruleCache        = new RuleCache()
+  val privadoInput     = PrivadoInput(limitArgExpansionDataflows = 20)
```

Comment:
Now without explicitly passing `limitArgExpansionDataflows` test case should pass, or is it not getting passed?

Reply:
Now working without having `limitArgExpansionDataflows`. Addressed here: https://github.com/Privado-Inc/privado-core/pull/779/commits/4dc0d5ae6dba8ffc3b56b0dd6d716316799fa167